litellm/tests/test_router.py

```diff
@@ -1888,20 +1888,25 @@ async def test_router_model_usage(mock_response):
                 raise e
 
 
+@pytest.mark.skip(reason="Check if this is causing ci/cd issues.")
 @pytest.mark.asyncio
 async def test_is_proxy_set():
     """
     Assert if proxy is set
     """
-    from httpcore import AsyncHTTPProxy
+    from httpx import AsyncHTTPTransport
 
     os.environ["HTTPS_PROXY"] = "https://proxy.example.com:8080"
```

Comment:
I think this could be an issue, because it sets/overwrites the proxy settings from the environment and does not reset those. 
I would suggest using a setup/teardown function to store/clean and restore the current environment settings. Or to use a fixture for that. 

What you basically have to do is:
1. read the current values of `HTTPS_PROXY`, `HTTP_PROXY` and `NO_PROXY` into a variable
2. clean all those values and only set `HTTP_PROXY` as you do in this line here
3. after the test (even if it fails), reset `HTTPS_PROXY`, `HTTP_PROXY` and `NO_PROXY` to the values, you have read and stored in point 1